pluginAndCore.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+
+usage() {
+    echo "usage: [[-b | --build] | [-r | --rebuild] | [-c | --clean]]"
+}
+
+perform() {
+    cd CockpitCore
+    echo ./gradlew $1
+    ./gradlew $1
+    echo
+    cd ..
+
+    cd CockpitPlugin
+    echo ./gradlew $1
+    ./gradlew $1
+    echo
+    cd ..
+}
+
+# http://linuxcommand.org/lc3_wss0120.php
+if [ "$1" = "" ]
+then
+    usage
+    exit 1
+fi
+
+# https://stackoverflow.com/a/16349776/4137318
+cd "${0%/*}"
+
+while [ "$1" != "" ]; do
```

Comment:
You've already checked if argument is not empty.
So, what's the reason of having this while loop?